Sources/Subs-Compat.php

```diff
@@ -8759,8 +8717,8 @@ function cleanRequest(): void
 	 *   in the request and their values are accepted request values.
 	 * - A scalar value, in which case no furthur checks are done.
 	 *
-	 * @param array $array
-	 * @param string $req_var request variable
+	 * @param array $value_list
+	 * @param string $var
```

Comment:
```suggestion
	 * @param array $value_list Acceptable values
	 * @param string $var Name of the $_REQUEST variable to check.
```

Should also replace `$req_val` with `$var` in the function description a few lines above.